docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+# Use postgres/example user/password credentials
```

Comment:
What is it for?
The credentials are: 
* db: testdb
* user: postgres
* password: postgres

I'd remove it